feed/templates/feed/base_template.html

```diff
@@ -0,0 +1,84 @@
+{% load static %}
```

Comment:
@paOmer Since this file is a core file in our application, and it is not related only to the `feed` app, I think it will be more organized to move this file to the `jobseeker/templates/jobseeker` folder. So the `jobseeker/templates/jobseeker` will be considered as the "global" folder of our application.

Reply:
Also, it is required to move the `feed/static/feed` folder to `jobseeker/static/jobseeker`

Reply:
See #33 about this.